include/npc.h

```diff
@@ -323,7 +323,7 @@ typedef struct Enemy {
     /* 0xB4 */ s8 aiPaused;
     /* 0xB5 */ s8 unk_B5;
     /* 0xB6 */ char unk_B6[2];
-    /* 0xB8 */ EvtScript* unk_B8; // some bytecode
+    /* 0xB8 */ EvtScript* rot; // some bytecode
```

Comment:
Remove comment
Also why is this rot? A rotation script?? Looks like a find replace accident considering the rest of the pr